web/src/components/netflow-traffic-parent.tsx

```diff
@@ -22,6 +23,7 @@ class NetflowTrafficParent extends React.Component<Props, State> {
     console.error('Netobserv plugin error. This is likely a bug, this error should be caught closer to its source.');
     console.error('Error:', error, 'errorInfo:', errorInfo);
     this.setState({ error: error });
+    clearLocalStorage();
```

Comment:
Are we sure we want to clear local storage on error caught? It seems a little too abrupt to me

Reply:
Would you prefer to have a button and suggest the user to do so ? :thinking: Or only manage this when we catch error in the hook but it doesn't cover all the cases ?

It only clears our app local storage. Not the entire browser.

Reply:
I think yes, asking explicit action from the user, with a button to clear storage would be cool.
Because errors are not necessarily related to corrupted storage, and clearing user preferences after a bug occurred, could be perceived as a double punishment (I get that it's only our own storage, but still, it contains some preferences the user may have set)

Reply:
Added a reset button to clear local storage + remove url parameters https://github.com/netobserv/network-observability-console-plugin/pull/212/commits/c179ec8aab76a543e72b02ec054db25f41d07d97